src/main/java/hudson/plugins/sshslaves/verifiers/MissingVerificationStrategyAdministrativeMonitor.java

```diff
@@ -55,4 +56,15 @@ public boolean isActivated() {
         return false;
     }
 
+    /**
+     * This method can be removed when the baseline is updated to 2.88
+     *
+     * @return If this version of the plugin is running on a Jenkins version where JENKINS-43786 is included.
+     */
+    public boolean isTheNewDesignAvailable() {
+        if (Jenkins.getVersion().isNewerThan(new VersionNumber("2.88"))) {
```

Comment:
`2.88` is only an example. it depends when https://github.com/jenkinsci/jenkins/pull/2857 is merged and released.